image_data_guide/02_structuring_data.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Structuring Your Data (Part 2/4)\n",
+    "Download | **Structure** | Preprocess | Train"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
```

Comment:
For all these notebooks, add a short blurb describing what this notebooks is for.

Reply:
Added a short blurb at beginning.